### PR TITLE
Added mod+shift+a shortcut for selecting an attachment in composer (fixes #92)

### LINKED ISSUE #92: [Feature Request] Add a keyboard shortcut for adding an attachment
I downloaded the latest release (yesterday). Lovely so far and a possible Mail killer for me but:

- No keyboard shortcut for adding an attachment in compose view(?) In apple mail this is Cmd+Shift+A
- On OSX clicking the "Edit Custom Shortcuts" button does nothing (although it does move the view of the prefs page by about 10 pixels)

### PR DESCRIPTION
**Pull request by dbhowell**
I know @nickolasclarke has already added a PR for #92 but I thought I'd add mine to compare.

Feel free to close mine if it isn't useful.

### CODE REVIEW

File: app/keymaps/base.json [hunk 1]
```diff
@@ -57,5 +57,7 @@
   "contenteditable:outdent": "mod+[",
   "contenteditable:indent": "mod+]",
   "contenteditable:next-selection": "mod+\"",
-  "contenteditable:open-spelling-suggestions": "mod+m"
+  "contenteditable:open-spelling-suggestions": "mod+m",
+
```
Comment (nickolasclarke) [inline app/keymaps/base.json:61]: no need for this blank line. Keep it all together in a single block for `contenteditable`

File: app/internal_packages/preferences/lib/tabs/keymaps/displayed-keybindings.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ module.exports = [
       ['composer:focus-to', 'Focus the To field'],
       ['composer:show-and-focus-cc', 'Focus the Cc field'],
       ['composer:show-and-focus-bcc', 'Focus the Bcc field'],
+      ['composer:select-attachment', 'Select file attachment'],
```
Comment (nickolasclarke) [inline app/internal_packages/preferences/lib/tabs/keymaps/displayed-keybindings.js:34]: I'm not sure what style this project conforms to, but this hanging `,` isnt needed.

Comment (dbhowell) [reply app/internal_packages/preferences/lib/tabs/keymaps/displayed-keybindings.js:34]: Hanging commas are throughout the project, so I've kept them for consistency. I'm assuming Airbnb-based style guide?

File: app/keymaps/base.json (hunk 1 above)
Comment (dbhowell) [reply app/keymaps/base.json:61]: Again this is for consistency (all core, contenteditable seemed to be blocked together. But should this `"composer:select-attachment": "mod+shift+a"` be in https://github.com/Foundry376/Mailspring/blob/84643cb3c4b1dd281ec005cddc95ad302266ef31/app/internal_packages/composer/keymaps/composer.json ?

File: app/internal_packages/preferences/lib/tabs/keymaps/displayed-keybindings.js (hunk 2 above)
Comment (nickolasclarke) [reply app/internal_packages/preferences/lib/tabs/keymaps/displayed-keybindings.js:34]: ah, I must have missed that. When I glanced quickly I thought I didnt see them. You probably looked more thoroughly.

File: app/keymaps/base.json (hunk 1 above)
Comment (nickolasclarke) [reply app/keymaps/base.json:61]: heh you are right again. When I read this I read it as part of the contenteditable block. Derping it up!

**Comment by bengotow** [comment]
Hey folks—this looks great! Thanks for getting this written and thanks @nickolasclarke for the review! I think your version of this PR would work as well, but since this has been tested a bit more I'll merge in this one.

Also I'm not sure it's documented in the readme, but I've started using this JavaScript autoformatter called `prettier` (https://github.com/prettier/prettier) which basically force-formats everything to a specific JavaScript style when you save in your editor. I think it has added like a zillion trailing commas for me :-) Worth checking out if you write a lot of JS!

**Pull request merged**